src/fragments/lib/auth/ios/credential_handling/10_credential_handling.mdx

```diff
@@ -0,0 +1,7 @@
+## Credential handling
+
+Amplify Library for Swift securely handles credentials and user identity information. Customers do not need to store, refresh, or delete credentials themselves. Instead, Amplify Library handles these operations using platform capabilities such as [`ASWebAuthenticationSession`](https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession) for web-based OAuth flows, and [Keychain Services](https://developer.apple.com/documentation/security/keychain_services/) for storing and retrieving sensitive information. Only the host app (or defined app group) can access values that Amplify Library for Swift stores, and Amplify Library also clears credentials if it detects a configuration change such as a new Cognito UserPool or Cognito Identity Pool.
```

Comment:
Nit: can we change it to Amplify Swift library, rather than Amplify library for swift. We should do that in all sections of the paragrah.